docs/auto_aug/augmentations.rst

```diff
@@ -0,0 +1,259 @@
+
+Applies :meth:`nvidia.dali.fn.transforms.translation` with shape-relative offset in x-axis using
+:meth:`nvidia.dali.fn.warp_affine`.
+
+.. code-block::
+
+    @augmentation(mag_range=(0., 1.), randomly_negate=True, ...)
+    def translate_x(sample, rel_offset, shape, fill_value=128, interp_type=None)
+
+.. function:: translate_x_no_shape(sample, *, magnitude_bin=None, num_magnitude_bins=None, **kwargs)
+
+Applies :meth:`nvidia.dali.fn.transforms.translation` with absolute offset in x-axis using
+:meth:`nvidia.dali.fn.warp_affine`.
+
+.. code-block::
+
+    @augmentation(mag_range=(0, 250), randomly_negate=True, ...)
+    def translate_x_no_shape(sample, offset, fill_value=128, interp_type=None)
+
+.. function:: translate_y(sample, *, magnitude_bin=None, num_magnitude_bins=None, **kwargs)
+
+Applies :meth:`nvidia.dali.fn.transforms.translation` with shape-relative offset in y-axis using
+:meth:`nvidia.dali.fn.warp_affine`.
+
+.. code-block::
+
+    @augmentation(mag_range=(0., 1.), randomly_negate=True, ...)
+    def translate_y(sample, rel_offset, shape, fill_value=128, interp_type=None)
+
+.. function:: translate_y_no_shape(sample, *, magnitude_bin=None, num_magnitude_bins=None, **kwargs)
+
+Applies :meth:`nvidia.dali.fn.transforms.translation` with absolute offset in y-axis using
+:meth:`nvidia.dali.fn.warp_affine`.
+
+.. code-block::
+
+    @augmentation(mag_range=(0, 250), randomly_negate=True, ...)
+    def translate_y_no_shape(sample, offset, fill_value=128, interp_type=None)
+
+.. function:: rotate(sample, *, magnitude_bin=None, num_magnitude_bins=None, **kwargs)
+
+Rotates the image using :meth:`nvidia.dali.fn.rotate`.
+
+.. code-block::
+
+    @augmentation(mag_range=(0, 30), randomly_negate=True)
+    def rotate(sample, angle, fill_value=128, interp_type=None, rotate_keep_size=True)
+
+.. function:: brightness(sample, *, magnitude_bin=None, num_magnitude_bins=None, **kwargs)
+
+Adjusts the brightness with :meth:`nvidia.dali.fn.brightness`. The magnitude is mapped to a [0, 2]
+parameter range.
+
+.. code-block::
+
+    @augmentation(mag_range=(0, 0.9), randomly_negate=True, ...)
+    def brightness(sample, parameter)
+
+.. function:: contrast(sample, *, magnitude_bin=None, num_magnitude_bins=None, **kwargs)
+
+
+Adjusts the contrasts using a channel-weighted mean as a contrast center. The magnitude is mapped
+to a [0, 2] parameter range.
+
+.. code-block::
+
+    @augmentation(mag_range=(0, 0.9), randomly_negate=True, ...)
+    def contrast(sample, parameter)
+
+.. function:: color(sample, *, magnitude_bin=None, num_magnitude_bins=None, **kwargs)
+
+Adjusts the color with :meth:`nvidia.dali.fn.saturation`. The magnitude is mapped to a [0, 2]
+parameter range.
+
+.. code-block::
+
+    @augmentation(mag_range=(0, 0.9), randomly_negate=True, ...)
+    def color(sample, parameter)
+
+
+.. function:: sharpness(sample, *, magnitude_bin=None, num_magnitude_bins=None, **kwargs)
+
+The outputs correspond to PIL's ImageEnhance.Sharpness.
```

Comment:
I would either extend the description or add a link to PIL.

Reply:
done